screens/Home.js

```diff
@@ -66,13 +59,24 @@ const Home = () => {
       });
   }, []);
 
+    useEffect(() => {
+      const fetchuserData = async () => {
+        await firestore.collection("users").doc(user.uid).get()
+              .then(doc => setusersFullname(doc.data().name))
+              .catch(error => console.log(error))
+      }
+      
+      fetchuserData()
+    }, [])
+  
```

Comment:
The screen keeps refreshing and I think it might be because of this code

Reply:
Please check why the screen keeps refreshing

Reply:
the useEffect hook has no dependency as such will only run after the page has loaded and it doesn't seem to refresh on my end. But I'll check the code to see what might potentially be causing the problem